### DISCUSSION TITLE
difference between Settings\VisualFBEditor64.ini and VisualFBEditorX64.ini?

### DISCUSSION THREAD

**Discussion by retsyo** [XusinboyBekchanov/VisualFBEditor#265] [Q&A]
there are 3 `ini` files under `Settings`, 2 are labelled with `64`, what are the difference?
```
VisualFBEditor32.ini
VisualFBEditor64.ini
VisualFBEditorX64.ini
```

**Comment by XusinboyBekchanov** [comment]
VisualFBEditor32.ini - settings for 32 bit version
VisualFBEditor64.ini - settings for 64 bit version
VisualFBEditorX64.ini - settings for 64 bit Unix version

For 32-bit, 64-bit, for Windows, for Linux versions of VisualFBEditor there may be other suitable settings, for example, compilers, debuggers, help, and more. Therefore they are separated.
